src/schemaToPropTypes.js

```diff
@@ -199,7 +253,14 @@ const getPropTypes = (schemaName, schema) => {
  */
 const schemasReducer = (str, [schemaName, schema]) => {
 	const componentName = formatComponentName(schemaName);
-	return `${str}\nexport const ${componentName} = {\n${getPropTypes(schemaName, schema)}};\n`;
+	const isObjectDefinition = schema.type === 'object';
+	const propTypes = getPropTypes(schemaName, schema);
+
+	if (isObjectDefinition) {
+		return `${str}\nexport const ${componentName} = {\n${propTypes}};\n`;
+	}
+
+	return `${str}\nexport const ${componentName} = ${propTypes};\n`;
```

Comment:
Just a general thought: It might be wiser to use something like prettier on the output and not care so much about indentation and newlines and such in our own code.

In a recent project that did code generation, I did something like this:

```js
generatedCode += `export const ${variableName} = ${JSON.stringify(object)}\n`
const options = await prettier.resolveConfig('.prettierrc')
const formattedCode = prettier.format(generatedCode, options)
```

Which freed to code from handling all the styling related things. But this is just food for thought :)

Reply:
Again, Nice for this suggestion! Added it here: https://github.com/unic/openapi-proptypes-generator/issues/15 😄